templates/gatekeeper.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: gatekeeper
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: gatekeeper
+  annotations:
+    appversion.kubeaddons.mesosphere.io/gatekeeper: "3.0.4-beta.1"
+    docs.kubeaddons.mesosphere.io/gatekeeper: "https://github.com/open-policy-agent/gatekeeper/blob/master/README.md"
+    values.chart.helm.kubeaddons.mesosphere.io/gatekeeper: "https://raw.githubusercontent.com/mesosphere/charts/master/staging/gatekeeper/values.yaml"
```

Comment:
this needs to be the commit reflected in master instead of the `master` reference here

Reply:
Updated.